api4/plugin.go

```diff
@@ -383,3 +388,55 @@ func installPlugin(c *Context, w http.ResponseWriter, plugin io.ReadSeeker, forc
 	w.WriteHeader(http.StatusCreated)
 	w.Write([]byte(manifest.ToJson()))
 }
+
+func setFirstAdminVisitMarketplaceStatus(c *Context, w http.ResponseWriter, r *http.Request) {
+	auditRec := c.MakeAuditRecord("setFirstAdminVisitMarketplaceStatus", audit.Fail)
+	defer c.LogAuditRec(auditRec)
+	c.LogAudit("attempt")
+
+	if !c.App.SessionHasPermissionTo(*c.App.Session(), model.PERMISSION_MANAGE_SYSTEM) {
+		c.SetPermissionError(model.PERMISSION_MANAGE_SYSTEM)
+		return
+	}
+
+	if err := c.App.Srv().Store.System().SaveOrUpdate(&model.System{
+		Name:  model.SYSTEM_FIRST_ADMIN_VISIT_MARKETPLACE,
+		Value: strconv.FormatBool(true),
```

Comment:
```suggestion
		Value: "true",
```

No need to convert a static value.